src/System.Net.NameResolution/src/System.Net.NameResolution.csproj

```diff
@@ -54,6 +54,8 @@
   </ItemGroup>
   <ItemGroup Condition=" '$(TargetsWindows)' == 'true'">
     <Compile Include="System\Net\NameResolutionPal.Windows.cs" />
+    <Compile Include="System\Net\NameResolutionPal.Windows.Win32.cs" Condition="'$(TargetGroup)' != 'uap'"/>
```

Comment:
Nit: There is no file with `Windows.Win32.cs` suffix today. Everything it using just `.Win32.cs`.

Reply:
Fixed